playwright.config.ts

```diff
@@ -47,18 +47,85 @@ export default defineConfig({
       : {}),
     testIdAttribute: 'data-ouia-component-id',
     trace: 'on',
-    screenshot: 'off',
+    screenshot: 'on',
     video: 'on',
   },
   projects: [
     { name: 'setup', testMatch: /.*\.setup\.ts/ },
-    {
-      name: 'chromium',
-      use: {
-        ...devices['Desktop Chrome'],
-        storageState: '.auth/user.json',
-      },
-      dependencies: ['setup'],
-    },
+    ...(process.env.INTEGRATION
+      ? [
+          {
+            name: 'API',
+            grepInvert: process.env.PROD
+              ? [/preview-only/, /switch-to-preview/, /local-only/]
```

Comment:
So we will need to specify all integration tests here? Can we maybe create standard that integration tests for example start with "integration-" or so?